doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -85,12 +85,14 @@ The Requirements Exceptions lists the Reference Model (RM) requirements and/or R
 ## 1.4 Scope
 
 The scope of this particular Reference Architecture can be described as follows (the capabilities themselves will be listed and described in subsequent chapters), also shown in Figure 1-1:
+
 - Kubernetes capabilities required to conform to the Reference Model requirements
 - Support for CNFs that consist wholly of containers
 - Support for CNFs that consist partly of containers and partly of VMs, both of which will be orchestrated by Kubernetes
+- **Kubernetes Cluster lifecycle management**: including Cluster creation/upgrade/scaling/deletion, and node customisation due to workload requirements. **Note**: *detailed requirements and component specification of cluster LCM are out of scope for this release.*
```

Comment:
I think line 27 needs updating too (I can't suggest updates as it wasn't changed in your commit):

> The Kubernetes Reference Architecture will be used to determine a Kubernetes Reference Implementation. The Kubernetes Reference Implementation would then also be used to test and validate the supportability and compatibility with Kubernetes-based Network Function workloads of interest to the Anuket community. It is expected the Kubernetes Reference Architecture, Reference Implementation, and Reference Conformance will be developed in parallel to OVP Phase 2, which is building on the work already in place for OpenStack implementations. The intention is to expand as much of the existing test frameworks to be used for the verification and conformance testing of Kubernetes-based workloads.

As well as "used to test and validate the supportability and compatibility with Kubernetes-based Network Function workloads" we probably need to extend to something like, "used to test and validate the supportability and compatibility with Kubernetes-based Network Function workloads, and lifecycle management of Kubernetes clusters".

As well as "used for the verification and conformance testing of Kubernetes-based workloads" we probably need to extend to something like, "used for the verification and conformance testing of Kubernetes-based workloads, and Kubernetes cluster lifecycle management".

Just to make sure we're capturing the impact this will have on RI2/RC2.

Reply:
added in https://github.com/cntt-n/CNTT/commit/8612b9b77d0214d75400b59e6939e5292ef0554e